eve/main.yml

```diff
@@ -189,12 +189,25 @@ stages:
     worker:
       type: local
     steps:
+    # Checking weither the content of the branch is only documentation related.
```

Comment:
weither => whether

Reply:
hahaha found a way to make different mistakes on the same word. Thanks !